docs/release-notes.rst

```diff
@@ -3959,10 +3944,8 @@ Version 0.12.7
    checkpoint, and terminate the trial.
 
 -  Add support for Estimator callbacks that stop training early, including the official
-   `stop_if_no_decrease_hook
-   <https://www.tensorflow.org/api_docs/python/tf/estimator/experimental/stop_if_no_decrease_hook>`__.
```

Comment:
If we're still using this feature, it's here: `https://www.tensorflow.org/versions/r2.11/api_docs/python/tf/estimator/experimental/stop_if_no_decrease_hook`, which is in line with the current verison of TensorFlow that we support.